### ISSUE TITLE
Button focus outline color

### ISSUE THREAD

**Issue by ysds** [twbs/bootstrap#23822]
I prefer the button's focus outline color is different from the background(border) color. Because the outline can not be seen in the case of a solid button group (`btn btn-primary` etc.).

Current:
![image](https://user-images.githubusercontent.com/4065765/30011038-6488d8fa-916f-11e7-84c4-6ac7bac14af7.png)

Lighter example:
![image](https://user-images.githubusercontent.com/4065765/30011075-99ba649e-916f-11e7-9fec-41e9b44842a5.png)

**Comment by andresgalante** [comment]
Focus colors are always variables. For example for btns:
https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L311-L312

You can always change them and recompile if you want.

**Comment by ysds** [comment]
Yes. But I think it should be visible by default.

**Comment by mdo** [comment]
Unsure how best to proceed on this... perhaps we saturate or tint the focus shadow instead of using a regualr alpha-transparent theme color?

**Comment by ysds** [comment]
@mdo Please check the PR. But it not match to outline's focus color. It is better to have the same color for consistency.
